.buildkite/x-pack/pipeline.xpack.dockerlogbeat.yml

```diff
@@ -22,34 +27,67 @@ env:
   PACKAGING_ARM_PLATFORMS: "linux/arm64"
   PACKAGING_PLATFORMS: "+all linux/amd64 linux/arm64 windows/amd64 darwin/amd64 darwin/arm64"
```

Comment:
Nit: Maybe move this to the packaging step since only one reference exists. The same applies to the `PACKAGING_ARM_PLATFORMS.`